### ISSUE TITLE
Negotiable Factors

### ISSUE THREAD

**Issue by jordanx9** [davisgibson/davisgibson#17]
This should help people to skip negotiations. Some information that is given on the title may be negotiable for the buyer and I think we should make that known to them. For example, if there are Liens on the title, we would list that lien in the Negotiable Factors section and the cost to clear it. After we have that cost, we could display what the new cost of the house would be after subtracting the Liens from it (Assuming the buyer is willing to pay it). 

Basically three prices:
1) The price the seller set, this could be an open bid
2) The new price when all Negotiable factors are included
3) The "Buy Now" price the seller set, meaning if this price is paid, all other bids are canceled and the house is registered to that buyer

**Comment by jordanx9** [comment]
It could also just be 2 prices to start, the Buy Now and Negotiable Factors, the bid could be added in later
